app/model/repository/BaseRepository.php

```diff
@@ -36,6 +36,11 @@ public function __construct(EntityManagerInterface $em, $entityType)
         $this->repository = $repository;
     }
 
+    public function getEntityManagerForDebugging(): EntityManagerInterface
```

Comment:
Weird method... should it stay?

Reply:
Yes, it might be helpful for future debugging sessions...